organization/mission.md

```diff
@@ -26,10 +26,10 @@ The current priorities of the organization are set
 by the [board of directors](governance-model.md)
 and reviewed every meeting.
 
-On [4 August 2022](board-of-directors-meetings/2022-08-04-minutes.md),
+On [11 January 2023](board-of-directors-meetings/2023-01-11-minutes.md),
```

Comment:
This link seems wrong, as those minutes say: "[Current priorities](https://about.publiccode.net/organization/mission.html#current-priorities) remain unchanged"

Reply:
I would say that's an oversight in the minutes, as above it does contain a lot of information about chapterization. 

```
      1. Decision: New role description for Boris:
         The Head of EU organization will further the mission with a focus on developing the work of the Foundation for Public Code in an EU context by setting up a local, European public code organization/foundation with the goal of “to support European Union public sector and the public sector of its member states in collaboratively developing open software and policy.”
   1. Future structure, US entity
      1. Decision: initiate the setup and operation of an independent US foundation ('option 2')
      1. Action: Eric to update Ben on runway and burnrate estimates
      1. Action: Ben to propose a cost cap for operation
   1. Future structure, existing entity
      1. Action: Boris and Eric to follow up on the gifts documentation and that they are correctly reported
      1. Action: Boris and Eric to explore possibility of restructuring current organization as an umbrella organization for chapters
   1. Future structure, EU entity
      1. Action: Boris to formulate a plan

```

Yes, it's not *incorrect* that chapterisation falls under `align with the needs of public organizations`, but I do think it would be much better to have this clearly stated under priorities, as it is a current priority.

Reply:
@Ainali what should we do there?

Reply:
I don't know. The clear way to solve this would be to either make an amendment to the minutes or to link to newer minutes where the priorities unambiguously did change.
It feels odd to leave this contradiction up to the reader to resolve by reading all other lines of the minutes and also draw the conclusion that there was an oversight in the minutes.

Reply:
Agreed - will follow up on this.